diff --git a/kernel/build/config-amd64 b/kernel/build/config-amd64
index c5cc42a..c6e0e0d 100644
--- a/kernel/build/config-amd64
+++ b/kernel/build/config-amd64
@@ -3991,10 +3991,11 @@ CONFIG_THERMAL_DEFAULT_GOV_STEP_WISE=y
 CONFIG_THERMAL_GOV_STEP_WISE=y
 # CONFIG_THERMAL_GOV_BANG_BANG is not set
 CONFIG_THERMAL_GOV_USER_SPACE=y
+# CONFIG_DEVFREQ_THERMAL is not set
 # CONFIG_PCIE_THERMAL is not set
 # CONFIG_THERMAL_EMULATION is not set
 
-@@ -5768,7 +5769,12 @@ CONFIG_MMC_CQHCI=y
+@@ -5768,7 +5769,13 @@ CONFIG_MMC_CQHCI=y
  # CONFIG_MMC_TOSHIBA_PCI is not set
  # CONFIG_MMC_MTK is not set
  CONFIG_MMC_SDHCI_XENON=m
-# CONFIG_SCSI_UFSHCD is not set
+CONFIG_SCSI_UFSHCD=y
+CONFIG_SCSI_UFS_BSG=y
+CONFIG_SCSI_UFS_HWMON=y
+CONFIG_SCSI_UFSHCD_PCI=y
+# CONFIG_SCSI_UFS_DWC_TC_PCI is not set
+# CONFIG_SCSI_UFSHCD_PLATFORM is not set
  # CONFIG_MEMSTICK is not set
  CONFIG_NEW_LEDS=y
  CONFIG_LEDS_CLASS=y
@@ -6437,7 +6438,21 @@ CONFIG_RPMSG_VIRTIO=y
  # end of Qualcomm PM Domains
  # end of PM Domains
  
-# CONFIG_PM_DEVFREQ is not set
+CONFIG_PM_DEVFREQ=y
+
+#
+# DEVFREQ Governors
+#
+CONFIG_DEVFREQ_GOV_SIMPLE_ONDEMAND=y
+# CONFIG_DEVFREQ_GOV_PERFORMANCE is not set
+# CONFIG_DEVFREQ_GOV_POWERSAVE is not set
+# CONFIG_DEVFREQ_GOV_USERSPACE is not set
+# CONFIG_DEVFREQ_GOV_PASSIVE is not set
+
+#
+# DEVFREQ Drivers
+#
+# CONFIG_PM_DEVFREQ_EVENT is not set
  # CONFIG_EXTCON is not set
  # CONFIG_MEMORY is not set
  # CONFIG_IIO is not set
@@ -6530,6 +6545,7 @@ CONFIG_NVMEM_SYSFS=y
  
  # CONFIG_FPGA is not set
  # CONFIG_TEE is not set
+CONFIG_PM_OPP=y
  # CONFIG_SIOX is not set
  # CONFIG_SLIMBUS is not set
  # CONFIG_INTERCONNECT is not set
